### DISCUSSION TITLE
ODBC Driver : display version/release number

### DISCUSSION THREAD

**Discussion by simonaubertbd** [duckdb/duckdb#5054] [Ideas]
### What happens?

I would like to know the version of my ODBC driver when connecting/configuring, etc.

### To Reproduce

Absent from both windows :
![image](https://user-images.githubusercontent.com/46959192/197139599-0fb19d9c-14f8-4bd4-9df7-a5de3abe0d7e.png)
![image](https://user-images.githubusercontent.com/46959192/197139747-0c83f912-0562-47dd-b877-7471cbcb493b.png)
(screenshot is in french but it means :"no version available"

### OS:

Windows 10

### DuckDB Version:

0.5.1

### DuckDB Client:

ODBC

### Full Name:

Simon AUBERT

### Affiliation:

Business Decision (Orange Business Services)

### Have you tried this on the latest `master` branch?

- [X] I agree

### Have you tried the steps to reproduce? Do they include all relevant data and configuration? Does the issue you report still appear there?

- [X] I agree

**Comment by hannes** [comment]
Again, happy to review a PR

**Reply by simonaubertbd** [reply]
From what I understand, I should add a file like this one
https://github.com/MonetDB/MonetDB/blob/master/clients/odbc/driver/driver.rc

However what I don't know is what else to edit to make it work.
